pydis_site/apps/api/viewsets/bot/infraction.py

```diff
@@ -38,6 +39,8 @@ class InfractionViewSet(
     - **active** `bool`: whether the infraction is still active
     - **actor__id** `int`: snowflake of the user which applied the infraction
     - **hidden** `bool`: whether the infraction is a shadow infraction
+    - **limit** `int`: default limit is 100
+    - **offset** `int`: default is 0
```

Comment:
The documentation should state what these actually do in addition to their defaults.

Reply:
I tried to resolve all the required changes. Please take a look.